budget app backend/models/piggybank.js

```diff
@@ -20,6 +20,6 @@ const piggyBankSchema = new mongoose.Schema({
     timestamps: true
 });
 piggyBankSchema.post("save", function (doc) {
-    console.log(`${doc.email} has been saved`)
+    console.log(`${doc.title} has been saved`)
 })
-module.exports = mongoose.model("Users", piggyBankSchema)
\ No newline at end of file
+module.exports = mongoose.model("Piggybank", piggyBankSchema)
```

Comment:
What is Piggybank?

Reply:
It is a feature that is out of MVP.  Users should be able to save some of their money in piggybank.